src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -71,7 +71,6 @@
 /* Replication model to use when creating distributed tables */
 int ReplicationModel = REPLICATION_MODEL_COORDINATOR;
 
-
```

Comment:
Why did we remove this blank line? Could you run citus_indent  before pushing changes? It will fix lots of issues like this.